lib/npo.src.js

```diff
@@ -189,8 +244,17 @@
 		if (self.def) {
 			self = self.def;
 		}
-
 		self.msg = msg;
+		if (!nextShouldNotEmit) {
+			schedule(function (){
+				if (!hasCatch(this.chain)){
+					emit('unhandledRejection', {promise: this.promise, reason: this.msg});
+					this.unhandled = true;
+				}
+			}, self);
+		} else {
```

Comment:
code style in this file has `else` on its own line, not "cuddled" with the previous `}`.